index.html

```diff
@@ -17,10 +17,57 @@
   </head>
   <body>
     <header>
-      <div class="container"></div>
+      <div class="container">
+        <div class="column-full">
+          <h1>Code Journal</h1>
+        </div>
+      </div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
+      <div data-view="entry-form">
+        <form id="journal-form">
+          <div class="row">
+            <div class="column-full">
+              <h1>New Entry</h1>
+            </div>
+          </div>
+          <div class="row">
+            <div class="column-half">
+              <img
+                id="image"
+                alt="Blank image"
+                src="images/placeholder-image-square.jpg" />
+            </div>
+            <div class="column-half">
+              <div class="row">
+                <p>Title</p>
+              </div>
+              <div class="row">
+                <input type="text" id="title" name="title" required />
+              </div>
+              <div class="row">
+                <p>Photo URL</p>
+              </div>
+              <div class="row">
+                <input type="text" id="photo" name="photo" required />
+              </div>
```

Comment:
You shouldn't have all of these `row`s inside of a `column-half`. You should just have the content itself.

If you are trying to fix the problem of your content appearing on the same line, think about `block` vs `inline`. What can you give to your `input`,s to get them to go on a new line?